hathor/indexes/manager.py

```diff
@@ -270,7 +271,17 @@ def handle_contract_execution(self, tx: BaseTransaction) -> None:
                             from hathor.nanocontracts.runner.types import SyscallRecordType
                             assert syscall.type is SyscallRecordType.CREATE_TOKEN, syscall.type
                             assert syscall.token_name is not None and syscall.token_symbol is not None
-                            self.tokens.create_token_info(syscall.token_uid, syscall.token_name, syscall.token_symbol)
+                            # TODO-Raul: check with Gabriel with we should enforce a default value here,
+                            # TODO-Raul: since the SyscallUpdateTokensRecord already has an default value
+                            # TODO-Raul: for token_version or if we should only allow deposit in the moment
+                            token_version = TokenInfoVersion.DEPOSIT
+                            if syscall.token_version is not None:
+                                token_version = syscall.token_version
```

Comment:
No default value should be used here. The index should follow whatever it receives from the runner.

Reply:
Thanks!